docs/recipes/authentication.md

```diff
@@ -6,10 +6,10 @@ at how we would implement this in NGXS.
 First, let's define our state model and our actions:
 
 ```TS
-export class AuthStateModel {
-  token?: string;
-  username?: string;
-}
+export type AuthStateModel = null | {
```

Comment:
Probably better as an interface instead of a type.

Reply:
Nah, because of discriminated union :D